app/scripts/modules/core/src/presentation/HoverablePopover.tsx

```diff
@@ -142,15 +142,15 @@ export class HoverablePopover extends React.Component<IHoverablePopoverProps, IH
   }
 
   private refCallback(ref: any): void {
-    this.setState({ target: ref })
+    this.setState({ target: ref });
   }
 
   public render() {
     const PopoverContents = this.props.template;
     const PopoverRenderer = this.state.PopoverRenderer;
 
     return (
-      <span onMouseEnter={this.handleMouseEvent} onMouseLeave={this.handleMouseEvent} ref={this.refCallback}>
+      <g onMouseEnter={this.handleMouseEvent} onMouseLeave={this.handleMouseEvent} ref={this.refCallback}>
```

Comment:
why `g` @jrsquared ?

Reply:
I don't remember. `span` caused issues, `g` didn't. Probably should have added a comment around this...